staticd/static_nb_config.c

```diff
@@ -709,8 +718,19 @@ int routing_control_plane_protocols_control_plane_protocol_staticd_route_list_pa
 int routing_control_plane_protocols_control_plane_protocol_staticd_route_list_path_list_frr_nexthops_nexthop_onlink_modify(
 	struct nb_cb_modify_args *args)
 {
+	int nh_type;
+
 	switch (args->event) {
 	case NB_EV_VALIDATE:
+		nh_type = yang_dnode_get_enum(args->dnode, "../nh-type");
+		if (nh_type != STATIC_IPV4_GATEWAY_IFNAME
+		    || nh_type != STATIC_IPV6_GATEWAY_IFNAME) {
```

Comment:
Shouldn't this be `&&`?

Reply:
ack and resolved it right away in the pr.